src/main/resources/bin/windup

```diff
@@ -57,7 +57,7 @@ while [ "$1" != "" ] ; do
     WINDUP_DEBUG_ARGS="-Xdebug -Xnoagent -Djava.compiler=NONE -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=8000"
   fi
 
-  QUOTED_ARGS="$QUOTED_ARGS \"$1\""
+  QUOTED_ARGS="$QUOTED_ARGS $1"
```

Comment:
Rather than try too hard to use my brain as bash parser I pulled this in and tried:
time bin/windup --input ~/apps/example/jee-example-app-1.0.0.ear --output "/home/dansmit/windup-reports/test-bin-windup/path with spaces" --target eap

and got:
Output Path:/home/dansmit/windup-reports/test-bin-windup/path

on 2.7 I got:
Output Path:/home/dansmit/windup-reports/test-bin-windup/path with spaces
(and it wrote the reports there)

so something has changed.